CI: `TestVolumesMultiAttach` permafailing

```
2023-06-20T15:50:40.5902347Z === RUN   TestVolumesMultiAttach
2023-06-20T15:50:40.8292534Z     convenience.go:36: [1;31mFailure in volumes_test.go, line 85: unexpected error [0m[1;33m"Bad request with: [POST http://10.1.0.56/volume/v3/ee4787d881734edeb2e7f4d067860ae1/volumes], error message: {\"badRequest\": {\"code\": 400, \"message\": \"multiattach parameter has been removed. The default behavior is to use multiattach enabled volume types. Contact your administrator to create a multiattach enabled volume type and use it to create multiattach volumes.\"}}"[0m[1;31m[0m
2023-06-20T15:50:40.8293891Z --- FAIL: TestVolumesMultiAttach (0.24s
```

It seems to affect multiple versions of the blockstorage jobs.

--- a/acceptance/openstack/blockstorage/v3/blockstorage.go
+++ b/acceptance/openstack/blockstorage/v3/blockstorage.go
@@ -177,6 +177,36 @@ func CreateVolumeTypeNoExtraSpecs(t *testing.T, client *gophercloud.ServiceClien
 	return vt, nil
 }
 
+// CreateVolumeTypeMultiAttach will create a volume type with a random name and
+// extra specs for multi-attach. An error will be returned if the volume type was
+// unable to be created.
+func CreateVolumeTypeMultiAttach(t *testing.T, client *gophercloud.ServiceClient) (*volumetypes.VolumeType, error) {
+	name := tools.RandomString("ACPTTEST", 16)
+	description := "create_from_gophercloud"
+	t.Logf("Attempting to create volume type: %s", name)
+
+	createOpts := volumetypes.CreateOpts{
+		Name:        name,
+		ExtraSpecs:  map[string]string{"multiattach": "<is> True"},
+		Description: description,
+	}
+
+	vt, err := volumetypes.Create(client, createOpts).Extract()
+	if err != nil {
+		return nil, err
+	}
+
+	tools.PrintResource(t, vt)
+	th.AssertEquals(t, vt.IsPublic, true)
+	th.AssertEquals(t, vt.Name, name)
+	th.AssertEquals(t, vt.Description, description)
+	th.AssertEquals(t, vt.ExtraSpecs["multiattach"], "<is> True")
+
+	t.Logf("Successfully created volume type: %s", vt.ID)
+
+	return vt, nil
+}
+
 // CreatePrivateVolumeType will create a private volume type with a random
 // name and no extra specs. An error will be returned if the volume type was
 // unable to be created.
@@ -268,7 +298,7 @@ func DeleteVolumeType(t *testing.T, client *gophercloud.ServiceClient, vt *volum
 
 	err := volumetypes.Delete(client, vt.ID).ExtractErr()
 	if err != nil {
-		t.Fatalf("Unable to delete volume %s: %v", vt.ID, err)
+		t.Fatalf("Unable to delete volume type %s: %v", vt.ID, err)
 	}
 
 	t.Logf("Successfully deleted volume type: %s", vt.ID)

--- a/acceptance/openstack/blockstorage/v3/volumes_test.go
+++ b/acceptance/openstack/blockstorage/v3/volumes_test.go
@@ -67,10 +67,15 @@ func TestVolumes(t *testing.T) {
 }
 
 func TestVolumesMultiAttach(t *testing.T) {
+	clients.RequireAdmin(t)
 	clients.RequireLong(t)
 
 	client, err := clients.NewBlockStorageV3Client()
 	th.AssertNoErr(t, err)
+
+	vt, err := CreateVolumeTypeMultiAttach(t, client)
+	th.AssertNoErr(t, err)
+	defer DeleteVolumeType(t, client, vt)
 
 	volumeName := tools.RandomString("ACPTTEST", 16)
 
@@ -78,19 +83,17 @@ func TestVolumesMultiAttach(t *testing.T) {
 		Size:        1,
 		Name:        volumeName,
 		Description: "Testing creation of multiattach enabled volume",
-		Multiattach: true,
+		VolumeType:  vt.ID,
 	}
 
 	vol, err := volumes.Create(client, volOpts).Extract()
 	th.AssertNoErr(t, err)
+	defer DeleteVolume(t, client, vol)
 
 	err = volumes.WaitForStatus(client, vol.ID, "available", 60)
 	th.AssertNoErr(t, err)
 
 	th.AssertEquals(t, vol.Multiattach, true)
-
-	err = volumes.Delete(client, vol.ID, volumes.DeleteOpts{}).ExtractErr()
-	th.AssertNoErr(t, err)
 }
 
 func TestVolumesCascadeDelete(t *testing.T) {

